src/pages/subreddit/Subreddit.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import Header from "../../components/header/Header";
+import './Subreddit.css';
+import { ReactComponent as BackIcon } from '../../assets/back.svg';
+
+
+function Subreddit() {
+    const {id} = useParams();
+    const [catchError, setCatchError] = useState(null);
+    const [isLoading, setLoading] = useState(false);
+    const [details, setDetails] = useState({});
```

Comment:
Ik zou de default waarde hier `undefined` maken, dan wordt je check verderop simpeler. Tegelijkertijd is het dan een duidelijker verschil of het al wél of niet is opgehaald
```suggestion
    const [details, setDetails] = useState();
```